WordPress/src/main/java/org/wordpress/android/push/GCMMessageService.java

```diff
@@ -312,36 +312,12 @@ private static void bumpPushNotificationsAnalytics(Stat stat, Bundle noteBundle,
         }
     }
 
-    private static boolean canAddActionsToNotifications(Context context) {
```

Comment:
ready, pushed several commits addressing the most obvious issues thrown by code inspector